helpers/setup-spark-k8s.py

```diff
@@ -139,3 +154,10 @@ def setup_spark_conf_defaults(username: str, namespace: str) -> None:
         namespace = args.namespace or defaults['namespace']
         set_up_user(username, namespace, defaults)
         setup_spark_conf_defaults(username, namespace)
+    elif args.action == 'sa-conf-create':
+        utils.setup_kubernetes_secret(args.properties_file, args.service_account_name)
```

Comment:
What about creating the properties when we setup the service account ? We could provide some flags (as properties file or a list of --conf flags) to the "setup-spark-k8s service account" command